lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -25,6 +26,11 @@ export class BrowserUtils {
         } else {
             window.location.assign(urlNavigate);
         }
+
+        // To block code from running after navigation, this should not throw if navigation succeeds
+        setTimeout(() => {
+            throw BrowserAuthError.createNavigationFailedError();
```

Comment:
Will this end up in teh catch of `handleRedirectPromise`?

Reply:
Only if navigation does not occur